woocommerce/woocommerce-gateway-stripe
# Implement the new design for settings tab

Fixes #2740 

## Changes proposed in this Pull Request:
This PR implements the final design iteration of the settings tab. (A few things have been updated after #2815)
- `Debug mode` checkbox is moved back to the `Advanced settings` section.
- `Advanced settings` section is now expanded by default without the accordion.
- The heading `Account status` is removed and we are showing the email (if present) and the `Test mode` pill on the header as before.

## Testing instructions
- Go to the `Settings` tab of Stripe settings page.
- Check the `General` section. This section should not have the `Debug mode` checkbox.

![general](https://github.com/woocommerce/woocommerce-gateway-stripe/assets/33387139/17d89a6d-4d13-4ac4-9657-3d0506922d9b)

- Scroll down to the bottom. The `Advanced settings` section should be expanded by default (without the accordion). The `Debug mode` and `New checkout experience` checkboxes should be in this section.

![advanced](https://github.com/woocommerce/woocommerce-gateway-stripe/assets/33387139/7c498e8e-d35f-41b0-a36d-e9a7265cfa69)
